src/Serilog.Sinks.Elasticsearch/LoggerConfigurationElasticSearchExtensions.cs

```diff
@@ -89,7 +94,8 @@ public static class LoggerConfigurationElasticsearchExtensions
             string bufferBaseFilename = null,
             long? bufferFileSizeLimitBytes = null,
             long bufferLogShippingInterval = 5000,
-            string connectionGlobalHeaders = null)
+            string connectionGlobalHeaders = null,
+            LoggingLevelSwitch loggingLevelSwitch = null)
```

Comment:
In the other public APIs exposed by Serilog, [we call this `levelSwitch`](https://github.com/serilog/serilog/blob/dev/src/Serilog/Configuration/LoggerSinkConfiguration.cs#L71) - would be good  to be consistent (if this change is accepted).

Reply:
Good point. Will align.